src/components/IconV2/IconV2-story.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  iconAdd,
+  iconAddGlyph,
+  iconAddOutline,
+} from 'carbon-icons/dist/carbon-icons-list';
```

Comment:
One thing I noticed, should we keep the build dir consistent between the libraries? e.g. here it's `dist`, react does `lib`, etc.

Reply:
@joshblack Interesting point - I guess the icons package chose `dist` given there is not much code in it. Also changing the distribution directory would be (minor) breaking change. But feel free to go forward with it if you feel strongly about it!